rest-api/tools/import_organizations.py

```diff
@@ -0,0 +1,156 @@
+"""Imports awardees / HPOs, organizations, and sites into the database.
+
+Usage:
+  tools/import_organizations.sh [--account <USER>@pmi-ops.org --project <PROJECT>] \
+      --awardee_file <AWARDEE CSV> --organization_file <ORGANIZATION CSV> \
+      --site_file <SITE CSV> [--dry_run]
+
+  The CSV files originate from the Google Sheets found here:
+
+  https://docs.google.com/spreadsheets/d/1CcIGRV0Bd6BIz7PeuvrV6QDGDRQkp83CJUkWAl-fG58/edit
+"""
+
+import logging
+
+from dateutil.parser import parse
+from tools.csv_importer import CsvImporter
+from dao.hpo_dao import HPODao
```

Comment:
I don't suppose there's an API you could hit instead of going directly to the DB?

Reply:
There is not at present.

We had discussed whether it was worthwhile to add one. This is the only way we intend to write organization metadata, so adding an API around it seemed like extra work we could avoid. If we add one in the future, it hopefully shouldn't be too bad to transition to using it.